ci/pipeline.yml

```diff
@@ -16,11 +16,23 @@ steps:
       - "tmp/build/**/*"
       {{/artifacts}}
       - "bazel-testlogs/**/test.log"
+    env:
+      PLAIDML_DEVICE: llvm_cpu.0
+      PLAIDML_TARGET: llvm_cpu
 {{/variants}}
 
 
 # test phase
 
+  - label: ':linux:unit_tests/intel_gen'
+    agents:
+      PLAIDML_TARGET: intel_gen
+    command: |
+      python3 ci/buildkite.py unit linux_x86_64-opt intel_gen
+    env:
+      PLAIDML_TARGET: intel_gen
```

Comment:
Is this still needed? I thought we wanted to use command line args instead? Also I see the environment being set from the test runner.

Reply:
The environment variables set in the test runner come from plan.yml; I preserved that code in case there are environment variables that aren't set in pipeline.yml.

We still have tests that use the environment variables, and I don't think it hurts to set them.

Reply:
So how would the device be specified? Is that thru an environment variable? Let's say we have multiple devices on a machine and we want one agent to be responsible for one device.

Reply:
I think we'd tag the agent with the device, set the device constraint in pipeline.yml (possibly starting to construct the unit test block from plan.yml at that point), and then either pass it as an environment variable or pass it as a command-line parameter through buildkite.py.  (I prefer the latter, just to make things more explicit, but I don't have strong feelings about the device setting.)

We might want to do this as part of implementing device selection.